cmd/krel/cmd/ff.go

```diff
@@ -176,72 +160,47 @@ func runFf(opts *ffOptions) error {
 	}
 	log.Printf("last tag is: %s", masterTag)
 
-	// TODO: Rewrite using go-git
-	//       --dry-run appears to be unsupported for git push, so we shell out here.
-	if err := command.Execute("git push -q --dry-run", kgit.KubernetesGitHubAuthURL); err != nil {
-		return err
-	}
-
-	w, err := repo.Worktree()
+	releaseRev, err := repo.Head()
 	if err != nil {
 		return err
 	}
 
-	// ... checking out to commit
-	//Info("git checkout %s", commit)
-	remoteHash, err := repo.ResolveRevision(plumbing.Revision(fmt.Sprintf("%s/%s", remote, branch)))
-	if err != nil {
+	if err := repo.Pull(remoteMaster); err != nil {
 		return err
 	}
 
-	err = w.Checkout(&git.CheckoutOptions{
-		Hash:   plumbing.NewHash(remoteHash.String()),
-		Branch: plumbing.NewBranchReferenceName(branch),
-		Create: true,
-	})
-
+	headRev, err := repo.Head()
 	if err != nil {
 		return err
 	}
 
-	// TODO: Merge and update
-	if err := command.Execute("git merge -X ours", remoteMaster); err != nil {
```

Comment:
Any reason we're dropping the merge + merge strategy here?

Reply:
Yeah it was not supported by the go-git implementation. I switched to shelling out by running repo.Push, so it works now as before.